README.md

```diff
@@ -20,7 +20,8 @@ This is considered an intermediate learning resource and should typically be ref
 ## Examples <a name="Examples"></a>
 This repo contains examples in each language supported by Amazon Keyspaces. This is not exhaustive.
 
-- [Java Examples](https://github.com/aws-samples/aws-keyspaces-examples/tree/master/java)
+- [Java Examples](https://github.com/rohangupta/amazon-keyspaces-examples/tree/main/java)
```

Comment:
Whops these point to your fork and not to the main repository

Reply:
Oops, added the right links. Wanted to fix the original broken link.